### PR TITLE
Fix scanning on iPhone

### PR DESCRIPTION
**Pull request by miguelbrandao**

### CODE REVIEW

File: pages/sponsor/scanner.js [hunk 1]
```diff
@@ -14,33 +14,36 @@ function SponsorBadges() {
...
 
   const handleUUID = (uuid) => {
+    let feedback;
     giveBadge(uuid, "69420")
       .then((response) => {
         if (response.redeem) {
-          navigator.vibrate([20, 10, 20]);
-          setFeedback(FEEDBACK.SUCCESS);
+          navigator.vibrate([40, 20, 40]);
+          feedback = FEEDBACK.SUCCESS;
```
Comment (nelsonmestevao) [inline pages/sponsor/scanner.js:33]: Shouldn't be setFeedback?

File: pages/sponsor/scanner.js [hunk 2]
```diff
@@ -57,8 +60,8 @@ function SponsorBadges() {
           pauseRef={pauseRef}
           text={user.name}
           feedback={feedback}
-          showScanner={showScanner}
-          setScanner={setScanner}
+          showScanner={true}
```
Comment (nelsonmestevao) [inline pages/sponsor/scanner.js:63]: Is this correct?

Comment (miguelbrandao) [reply pages/sponsor/scanner.js:63]: For companies yes

**Review by nelsonmestevao — APPROVED** [review]
It works on my iPhone 🙌

**Pull request merged**